pkg/manifest/manifest.go

```diff
@@ -590,12 +581,41 @@ func (m *Manifest) SetDefaults() error {
 			}
 			m.Image = reference.FamiliarString(imageRef)
 		case reference.Digested:
-			return errors.New("invalid bundle tag format %q, must be an OCI image tag")
+			return errors.New("invalid bundle tag format, must be an OCI image tag")
 		}
 	}
+
 	return nil
 }
 
+// getDockerTagFromBundleRef returns the Docker tag portion of the bundle tag,
+// updating the manifest BundleTag value if it initially lacks a Docker tag
```

Comment:
I feel like this function is doing too much now that it is called `get*`. I wouldn't expect a function called get to also do an update.

It would make be preferable from a separation of responsibility standpoint if it just get the tag, and then have `SetInvocationImageFromBundleTag` continue to manage setting `m.Tag` like it did before.

Reply:
Good call; will update such that setting the manifest BundleTag value occurs in `SetInvocationImageFromBundleTag`.